Rmarkdown/Multiple_Ascending_Dose_PD_real_example.Rmd

```diff
@@ -106,11 +106,11 @@ dirs = list(
 
 ```
 
-
```

Comment:
A lot of these Rmd files have deleted or added spaces and that's it.  I'm just curious what's going on here, did you actually delete and add a bunch of blank lines from various files?  I think it's fine if you don't know what happened, I was just curious.